src/views/Player.vue

```diff
@@ -255,6 +281,10 @@ export default {
               this.plioQuestions[index].user_answer = this.answers[index];
               this.plioQuestions[index].state =
                 this.answers[index].length == 0 ? "notshown" : "answered";
+
+              if (this.plioQuestions[index].state == "answered") {
+                this.progressBarInfo['completionPercent'] += 1
+              }
```

Comment:
iterate through the answers of the previous session, if any question has been answered, increase the progress counter